src/App/PropertyLinks.cpp

```diff
@@ -1091,6 +1127,17 @@ DocumentObject *PropertyLinkList::find(const std::string &name, int *pindex) con
     return _lValueList[it->second];
 }
 
+DocumentObject *PropertyLinkList::find(const std::string &name, int *pindex) const {

```

Comment:
So there are now two `find` methods, one taking a `const char *`, and this new one taking a `std::string`, and the `string` version has a whole additional logic statement at the front? I'm not familiar with this part of the code, but that seems like a very suspect overload.

Reply:
Yeah, this is hideous.  Turns out the std::string method is called exactly five times - everything else uses the const char * version.  The string version has been renamed in #15501 to findUsingMap and comments added to explain what's happening here.  Rational, but was very opaque.